### ISSUE TITLE
File wont open after Sharing to another device

### ISSUE THREAD

**Issue by mujeeb272** [Sriharia/share-them#15]
i sent many images , video to another device through shareThem , But file wont display in another device . it shows file cant open

**Comment by VenkataLochanN** [comment]
Yeah, I even faced this issue. Try checking the file Location... It might be in android/data/(either app folder or Download Manager folder).. Just copy the file to your required location and it opens...  A file explorer(like ES) other than the stock one is recommended. 

I you used webshare, the fle extention gets an extra apostrophe(due to a small bug in nanoHTTP.java file) making it unavailable.. delete that char and rename it. It will work.
